java/src/main/java/org/rocksdb/ReadOptions.java

```diff
@@ -750,6 +750,37 @@ public ReadOptions setValueSizeSoftLimit(final long valueSizeSoftLimit) {
     return this;
   }
 
+  /**
+   * Experimental
+   * <p>
+   * If async_io is enabled, RocksDB will prefetch some of data asynchronously.
+   * RocksDB apply it if reads are sequential and its internal automatic
+   * prefetching.
+   * <p>
+   * Default: false
+   * @return true if async_io is enabled.
+   */
+  public boolean asyncIO() {
```

Comment:
@InfiniteNightmare If this is experimental it should have the `@Experimental` annotation. Could you add it please?

Reply:
Also Java convention dictates that this should be named `asyncIo` and not `asyncIO` please